src/test/scala/mesosphere/marathon/api/akkahttp/v2/AppsControllerTest.scala

```diff
@@ -1908,5 +1904,105 @@ class AppsControllerTest extends UnitTest with GroupCreation with ScalatestRoute
         header[Headers.`Marathon-Deployment-Id`] should not be 'empty
       }
     }
+
+    "Replace multiple existing applications using Patch" in new Fixture {
+      Given("An app and group")
+      val app1Id = PathId("/app1")
+      val app2Id = PathId("/app2")
+
+      val newApp1Cmd = "bla1"
+      val newApp2Cmd = "bla2"
+      val appDefs = Seq(
+        AppDefinition(id = app1Id, cmd = Some("cmd1")),
+        AppDefinition(id = app2Id, cmd = Some("cmd1"))
+      )
+
+      val rootGroup = createRootGroup(
+        appDefs.map { appDef =>
+        appDef.id -> appDef
+      }.toMap
+      )
+      val plan = DeploymentPlan(rootGroup, rootGroup)
+
+      groupManager.updateRoot(any, any, any, any, any) answers { args =>
+        val fn = args(1).asInstanceOf[RootGroup => RootGroup]
+        val updated = fn(rootGroup)
+        updated.app(app1Id).get.cmd.get shouldEqual newApp1Cmd
```

Comment:
Just be explicit. `newApp1Cmd` is used only here so why not state the string explicitly

`updated.app(app1Id).get.cmd.get shouldEqual "bla1"`

Reply:
Actually there is a json below where we use it too :)

Reply:
True. Still. I don't think we save much here :).

Reply:
If I inline it, someone will have to change code in two places instead of one. In my view it will make this test less developer-friendly.